Assets/LeapMotionModules/Interaction/Examples/Buttons/InteractionToggle.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+namespace Leap.Unity.UI.Interaction {
+
+  /// <summary>
+  /// A physics-enabled toggle. Toggling is triggered by physically pushing the toggle to its compressed position. 
+  /// </summary>
+  public class InteractionToggle : InteractionButton {
+
+    [Tooltip("The height that this button rests at; this value is a lerp in between the min and max height.")]
+    [Range(0f, 1f)]
+    public float ToggledRestingHeight = 0.5f;
+    [Space]
+    public bool Toggled = false;
+
```

Comment:
I think this class should have an OnToggle(bool) callback, shouldn't it?
Probably also OnToggleEnable() and OnToggleDisable() (OnEnable/OnDisable are taken by Unity :\ )